### ISSUE TITLE
Keywords can't be treated as functions

### ISSUE THREAD

**Issue by jogrms** [lantiga/ki#29]
```
ki require core
ki (:a {:a :b}) 
```
Results in
```
TypeError: keyword(...) is not a function
```

Example:
http://ki-lang.org/editor/editor.html#ki%20require%20core%0Aki%20(:a%20%7B:a%20:b%7D)%20%20%20%20%0A%20%20%20%20

**Comment by lantiga** [comment]
Yes, this is a known limitation, at least for now. Keywords cannot be evaluated in ki as they are in Clojure, you should use `get` in the meantime:
```
ki require core
return ki (get {:a :b} :a)
```

**Comment by jogrms** [comment]
Can we make the `keyword` function return a function?

**Comment by lantiga** [comment]
The `keyword` function currently returns a ClojureScript keyword object. We could indeed make `keyword` return a function and attach all properties of the keyword object to it (making it callable in a way), but this breaks things like `equals`, etc in mori. Any thoughts?
